### PR TITLE
Fix #19

### LINKED ISSUE #19: Metronome keeps resetting to 0 at certain attacks and such. Not sure what's causing it but it makes the metronome unusable.

### PR DESCRIPTION
**Pull request by Thource**

### CODE REVIEW

File: src/main/java/com/visualmetronome/VisualMetronomePlugin.java [hunk 1]
```diff
@@ -55,96 +55,60 @@ VisualMetronomeConfig provideConfig(ConfigManager configManager)
...
+                case 10:
+                    currentColor = config.getTick10Color();
+            }
+        }
     }
     @Subscribe
     public void onConfigChanged(ConfigChanged event)
     {
-        if (tickCounter > config.colorCycle())
+        if (!event.getGroup().equals("visualmetronome")) {
```
Comment (Thource) [inline src/main/java/com/visualmetronome/VisualMetronomePlugin.java:103]: The issue is fixed without this check, I added it in last to be sure of it, but your plugin should still check that the config change belongs to it

**Comment by vincent0955** [comment]
Hey, you actually changed the whole implementation along with fixing the issue, it looks like you didn't take into account that the tickCounter number is actually shown to the client if the feature is on. So simplifying the implementation like you did would mess up the showTick and showPlayerTick features.

I went ahead and made my own commit with the fix you made but did it without changing the plugin implementation, hope you don't mind. Thanks for fixing the issue.

**Comment by vincent0955** [comment]
https://github.com/vincent0955/Visual-metronome/commit/a4737dfd68d942236f288458c8d07883f63f30d2

**Comment by Thource** [comment]
Oops, I did check showTick and showPlayerTick but maybe it was 1 based and I made it 0 based? Glad it’s fixed anyway :)

**Comment by vincent0955** [comment]
Yeah that, and also if config.tickCount() was set to 1, it would just stay at 0 instead of cycling through the numbers when you simplified it to 1 switch statement; though I am now just realizing that it can be fixed by adding checks to see if (config.tickCount() == 1) in the overlay files to choose which variable to display.

**Pull request closed without merging**